README.md

```diff
@@ -45,6 +45,9 @@ that you set `g:sql_clib_path` or `vim.g.sql_clib_path` to where
 `libsqlite3.so` is located, and if that the case, pr or issue are welcomed to
 add to sql.nvim lookup paths for `libsqlite3.so`.
 
+If installing this plugin on `ubuntu` you might need to install `libsqlite3-dev`, in order for
```

Comment:
I was planning to rewrite this section to contain subsection for each operating system like 

```markdown 
#### Arch 
Commands to run to meet sql.nvim requirements

#### Ubuntu 
Commands to run to meet sql.nvim requirements

#### MacOs
Commands to run to meet sql.nvim requirements

#### Windows 
Commands to run to meet sql.nvim requirements

#### Neovim
Current installation instruction 
```

I think this will make it easier for new users to quickly setup sql.nvim.

Reply:
@akinsho I believe adding the above template, moving the current instruction  to `#### neovim` and adding commands to run `#### Ubuntu` would be sufficient for this pr, I'll do follow PR to add the rest.

Reply:
👍🏿 so should I add the blank ones in this PR or just `Ubuntu` and `Neovim`?

Reply:
Yep just leave them blank or coming soon.